user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -818,14 +897,49 @@ def __build_global_report_summary(self,
+
+        conversion_items_summary = {}
+        if self.ctxt.get_ctxt('cpuClusterProxy'):
+            cpu_cluster_info = self.ctxt.get_ctxt('cpuClusterProxy')
+            gpu_cluster_info = self.ctxt.get_ctxt('gpuClusterProxy')
+            if cpu_cluster_info is not None and gpu_cluster_info is not None:
+                if cpu_cluster_info.get_worker_node().instance_type == gpu_cluster_info.get_worker_node().instance_type:
+                    conversion_items_summary['all'] = cpu_cluster_info.get_worker_node().instance_type
+                else:
+                    conversion_items_summary['all'] = cpu_cluster_info.get_worker_node().instance_type + ' to ' \
+                                                       + gpu_cluster_info.get_worker_node().instance_type
+            else:
+                conversion_items_summary['all'] = cpu_cluster_info.get_worker_node().instance_type
+
+        gpu_cluster_info_per_app = self.ctxt.get_ctxt('gpuClusterInfoPerApp')
+        cpu_cluster_info_per_app = self.ctxt.get_ctxt('cpuClusterInfoPerApp')
+        if cpu_cluster_info_per_app is not None:
+            for app_id in cpu_cluster_info_per_app:
+                cpu_cluster_info = cpu_cluster_info_per_app[app_id]
+                gpu_cluster_info = gpu_cluster_info_per_app[app_id]
+                if cpu_cluster_info is not None and gpu_cluster_info is not None:
+                    cpu_instance_type = cpu_cluster_info.get_worker_node().instance_type
+                    gpu_instance_type = gpu_cluster_info.get_worker_node().instance_type
+                    if cpu_instance_type == gpu_instance_type:
+                        conversion_items_summary[app_id] = cpu_instance_type
+                    else:
+                        conversion_items_summary[app_id] = cpu_instance_type + ' to '\
+                                                           + gpu_instance_type
+
+        rapids_output_dir = self.ctxt.get_rapids_output_folder()
+        tunings_dir = FSUtil.build_path(rapids_output_dir,
+                                        self.ctxt.get_value('toolOutput', 'csv', 'tunings', 'subFolder'))
+
         return QualificationSummary(comments=report_comments,
                                     all_apps=apps_grouped_df,
                                     recommended_apps=recommended_apps,
                                     savings_report_flag=launch_savings_calc,
                                     df_result=df_final_result,
                                     irrelevant_speedups=speedups_irrelevant_flag,
                                     sections_generators=[self.__generate_mc_types_conversion_report],
-                                    top_candidates_flag=filter_top_candidate_enabled)
+                                    top_candidates_flag=filter_top_candidate_enabled,
+                                    conversion_items=conversion_items_summary,
```

Comment:
I'm not following hte comment, we don't want them straight as comments because we process the conversion_items below.  Maybe I"m missing something?

```
f 'all' in self.conversion_items:
                print_result = self.df_result
                print_result['Qualified Node Recommendation'] = self.conversion_items['all']
            else:
                # add the per app node conversions
                conversion_column_dict = {'App ID': list(self.conversion_items.keys()),
                                          'Qualified Node Recommendation': list(self.conversion_items.values())}
                conversion_df = pd.DataFrame.from_dict(conversion_column_dict)
                print_result = pd.merge(self.df_result, conversion_df, on=['App ID'], how='left')

```